docs/rules/naming-convention.md

```diff
@@ -179,6 +149,7 @@ The object is an array with all elements of the type `string`.
 Additional restrictions:
 
 * Minimum items: `1`
+* Unique items: `true`
 
 ## Resources
 
```

Comment:
Consider adding in a suggested rule config folks can just paste in as a starting point. What you use in `recommended` is probably sufficient.

Reply:
Added more examples